react-components/src/components/BundleProductOptions/checkbox.js

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ *
+ *    Copyright 2020 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+import React from 'react';
+import { array, shape, func, bool, number } from 'prop-types';
+import Price from '../Price';
+
+const Checkbox = props => {
+    const { item, customization, sortedOptions, handleSelectionChange } = props;
+
+    const onChange = event => {
+        const { checked, value } = event.target;
+        let newCustomization;
+
+        if (checked) {
+            newCustomization = customization;
```

Comment:
Are you sure this works as expected? You copy a reference to an array here, so any change made to `newCustomization` is also applied to `customization`. Probably better to us
```
newCustomization = [...customization];
```

Reply:
doesn't really matter. changing `customization` does not trigger rendering. and `newCustomization` is then applied to a function coming from the parent component which triggers a state change thus triggering re-rendering.
So, this works as expected but it's better to do a shallow copy to avoid any issues/confusion